features/topic.feature

```diff
@@ -0,0 +1,19 @@
+@topic
+Feature: Create, edit and delete a topic in the course on the HPI SchulCloud page
+
+	Background:
+		Given Teacher arrives on the Schul-Cloud homepage
+		Given Teacher logs in with email lehrer@schul-cloud.org and password Schulcloud1!
+		Given Teacher accepts data protection
+
+	@createTopic
+	Scenario Outline: <userRole> creates a new topic in course
+		When <userRole> goes to courses page
+		And <userRole> chooses course with name <courseName>
```

Comment:
We also never made the user role as an example, since the testcases differ when it comes to each role. Might be a nice idea, but I don't think we will do it in such a way in the future.